packages/suite/src/actions/suite/logActions.ts

```diff
@@ -85,13 +91,14 @@ export const getLog =
 
 export const reportToSentry = (error: any) => (dispatch: Dispatch, getState: GetState) => {
     const { analytics, wallet, suite } = getState();
+
     Sentry.withScope(scope => {
         scope.setUser({ id: analytics.instanceId });
         scope.setContext('suiteState', {
             device: redactDevice(suite.device) ?? null,
-            discovery: wallet.discovery,
+            discovery: prettifyLog(wallet.discovery.map(d => redactDiscovery(d))),
```

Comment:
nit: no need for cryptic name
```suggestion
            discovery: prettifyLog(wallet.discovery.map(redactDiscovery)),
```